cypress/integration/dryRunner/cardLog.js

```diff
@@ -0,0 +1,47 @@
+import s from './selectors'
+const DECK_ID =
+  'MU4xLDFOMiwxSTIsMU4zLDFONCwxTjUsMU42LDFONjIsMU42MywxTjgsMU4xMCwxTjEz'
+const HAND = ['N1', 'N3', 'I2', 'N10']
+
+describe('Dry-runner — Card Log', () => {
+  beforeEach(() => {
+    cy.visit(`/deck/${DECK_ID}/dry-run?mode=MANUAL`).drDrawHand(HAND)
+  })
```

Comment:
I think we could speed up the test a bit by visiting `before`, and then just operating on the same game. The first test would play 1 card, then the 2nd test would play another card, then the 3rd test would play 5 more cards.

Reply:
You're right